### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by bobb-Rob**
### Implement the design from the sneak peek wireframe
- Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="user_card">
+  <div class="user_img my_border">
+    <img src="<%= user.photo %>" width="100" height="100" alt="Profile picture">
+  </div>
+  <div class="user_info my_border">
+    <h2 class="user_name">
+      <%= user.name %>
```
Comment (FrederickMih) [inline app/views/users/_user.html.erb:7]: - [**Optional**] Based on the requirements of this project, clicking on each user takes you to their show page. Though I can still access the show page at http://localhost:3000/users/1, kindly make sure that this should be a clickable link to improve the user experience.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,15 @@
...
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+  <div>    
+    <% @user.last_three_posts.each do |post| %>
+      <%= render "posts/post", post: post %>
+    <% end %>    
+  </div>
+  <%= link_to user_posts_path(@user.id) do %>
+    <%= button_tag 'See all posts', type: 'button', class: "see_all_btn btn" %>
```
Comment (FrederickMih) [inline app/views/users/show.html.erb:13]: - [Optional] On the wireframe, this button is centralized :smile:

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1 +1,15 @@
...
\ No newline at end of file
+<div>
+  <%=render 'user', user: @user %>
+  <div class="user_bio my_border">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+  <div>    
+    <% @user.last_three_posts.each do |post| %>
+      <%= render "posts/post", post: post %>
```
Comment (FrederickMih) [inline app/views/users/show.html.erb:9]: - kodus for making use of partials :+1:

**Review by FrederickMih — APPROVED** [review]
Hi @bobb-Rob , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :green_circle: 

- Descriptive commit messages are written in imperative tense :+1: 
- Views are all implemented and match the wireframes :+1: 
- Pull Request has a title and summary :+1: 
- Linters checks are OK :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/_user.html.erb (hunk 1 above)
Comment (bobb-Rob) [reply app/views/users/_user.html.erb:7]: Alright, this is well noted

File: app/views/users/show.html.erb (hunk 2 above)
Comment (bobb-Rob) [reply app/views/users/show.html.erb:13]: Thanks, I will fix this

**Pull request merged**